artem-serbin/README.md

```diff
@@ -0,0 +1,45 @@
+Google Maps
+===========
+
+
+
+Google Maps is a widely used web mapping service developed by Google. It provides users with detailed information about geographical locations and allows them to explore various places around the world. This README file provides an overview of Google Maps and its features.
+
+## Features
```

Comment:
You should put blank lines before and after a heading for compatibility (see https://www.markdownguide.org/basic-syntax#heading-best-practices). Also, as per MS Style Guide (https://learn.microsoft.com/en-us/style-guide/top-10-tips-style-voice#when-in-doubt-dont-capitalize), 'default to sentence-style capitalization—capitalize only the first word of a heading or phrase and any proper nouns or names.'. For example, 'Google Maps' is a proper name, while in the 'Search Functionality' phrase 2nd word shouldn't be capitalized. Please use this rule in the description below where applicable.